bin/tgsplit

```diff
@@ -216,7 +216,7 @@ def split_bkg_tge2( infile, clobber ):
 
     # save values
     tab.get_column("counts").values = bgcts
-    tab.get_column("backscal").value = bgscl
+    tab.get_column("backscal").values = bgscl
```

Comment:
So just so I don't have to think, the `get_column(foo).value` call succeeded, but filled in a slot which doesn't mean anything to Crates and so left the actual data alone. Now you have populated the field Crates wants you to populate, the correct background is being copied over?